Azure/sonic-buildimage
# Upgrade mellanox platform containers (syncd / saiserver / syncd-rpc) and pmon to bullseye

Fixes #9279 

#### Why I did it

Part of larger effort to move all SONiC systems to bullseye

#### How I did it

1. Update container makefiles with correct dependencies
2. Update container Dockerfile with correct base image
3. Update container Dockerfile with correct apt dependencies
4. Update any other makefiles with dependencies to remove python2 support
5. Minor changes to support bullseye / python3

#### How to verify it

Run regression on the switch:
1. Verify PTF community tests work
2. Verify syncd runs and all ports come up / pass traffic
3. Verify all platform tests succeed

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
Upgrade mellanox platform containers (syncd / saiserver / syncd-rpc) and pmon to bullseye

#### A picture of a cute animal (not mandatory but encouraged)
![Bullseye_spelling_Andy](https://user-images.githubusercontent.com/5898707/163441152-8c1f8d33-8591-41d5-80b0-0768289a57ec.jpg)
